src/components/LeafletMap.js

```diff
@@ -20,29 +20,52 @@ const Wrapper = styled.div`
   width: ${props => props.width};
   height: ${props => props.height};
 `;
-const myCustomColour = '#A53434';
+const defaultMarkerColor = '#A53434';
```

Comment:
I'd love for you to figure out a way to abstract out this value to the styled object so that there's one source of truth. 

But this isn't a blocker. I won't block the PR for this.